vendor/bat-native-ads/src/bat/ads/internal/bundle.cc

```diff
@@ -173,6 +174,17 @@ std::unique_ptr<BundleState> Bundle::GenerateFromCatalog(
         }
       }
 
+      // Conversions
+      for (const auto& conversion : creative_set.conversions) {
+        AdConversionInfo ad_conversion_info;
```

Comment:
Can we initialize `ad_conversion_info` with copy constructor here?

Reply:
Unfortunately not

Reply:
Fixed as refactored code to use a single data structure